packages/models-library/src/models_library/api_schemas_webserver/product.py

```diff
@@ -88,7 +88,7 @@ class GenerateInvitation(InputSchema):
 
 class InvitationGenerated(OutputSchema):
     product_name: ProductName
-    issuer: LowerCaseEmailStr
+    issuer: str
     guest: LowerCaseEmailStr
```

Comment:
questions here: 
1. is the issuer both a pseudo and an email? cause the second example looks like an email.
2. what is the difference with the guest field?

Reply:
it's a truncated email field. And it's just a reference. In some cases it might even be empty. I've checked with PC and the intention is correct.

The guest is the person being invited, the issuer is just an email or piece of email or a string reference of the Oetiker's system issuing invitations.